locksmith/__tests__/app.test.ts

```diff
@@ -76,16 +76,15 @@ describe('Requesting lock details', () => {
     test('should return an OK status code', async () => {
       expect.assertions(1)
       let response = await request(app).get(`/lock/${testLockDetails.address}`)
-      expect(response.statusCode).toBe(200)
+      expect(response.status).toBe(200)
```

Comment:
This was interesting. Once I converted this file to TS and installed the typings for `supertest`, it complained that `statusCode` was not a valid property. Changing it to  `status` satisfied the compiler, and the tests still pass so 🤷‍♂️